### PR TITLE
implement The Class Act latest wording

### LINKED ISSUE #6347: Class act leaves a card in the stack when drawing your entire deck (unless it's one card)
I only noticed because I've been playing a lot of Obelus.

If you draw cards equal to (or greater than) your deck size, you have to bottom one, then it stays. I think we can probably skip the class act trigger altogether if `cards-drawn >= cards-in-deck` (right now we skip it if there's exactly one card in deck).

### PR DESCRIPTION
**Pull request by francescopellegrini**
Fixes #6347 and #6107

### CODE REVIEW

**Comment by NoahTheDuke** [comment]
When did the wording change again??? How many times will we have to fix her? 😭

**Comment by francescopellegrini** [comment]
> When did the wording change again??? How many times will we have to fix her? 😭

Current Official Text:

When your discard phase ends, if you installed this resource this turn, draw 4 cards.
[interrupt] → The first time each turn you would draw any number of cards, look at the
top X cards of your stack. Add 1 of those cards to the bottom of your stack. X is equal
to the number of cards you will draw plus 1.

File: src/clj/game/core/drawing.clj [hunk 1]
```diff
@@ -46,50 +46,50 @@
...
-                   (doseq [c (get-set-aside state side set-aside-eid)]
-                     (move state side c :hand))
-                   (effect-completed state side eid))
-                 (let [draw-event (if (= side :corp) :corp-draw :runner-draw)]
-                   (swap! state update-in [side :register :currently-drawing] conj drawn)
-                   (queue-event state draw-event {:cards drawn
-                                                  :count drawn-count})
-                   (wait-for
-                     (checkpoint state nil (make-eid state eid) nil)
+       (wait-for (trigger-event-sync state side (make-eid state eid) (if (= side :corp) :pre-corp-draw :pre-runner-draw) n)
```
Comment (NoahTheDuke) [inline src/clj/game/core/drawing.clj:49]:
```suggestion
       (wait-for (trigger-event-simult state side (make-eid state eid) (if (= side :corp) :pre-corp-draw :pre-runner-draw) n)
```

File: src/clj/game/core/drawing.clj [hunk 2]
```diff
@@ -46,50 +46,50 @@
    (if (zero? n)
      (effect-completed state side eid)
      (do
```
Comment (NoahTheDuke) [inline src/clj/game/core/drawing.clj:48]: This should be removed because `wait-for` has an implicit `do` already.

**Pull request merged**